NetLicensingClient/LicenseeService.cs

```diff
@@ -99,7 +99,7 @@ public static ValidationResult validate(Context context, String number, String p
         /// In the case of multiple product modules validation, required parameters indexes will be added automatically.
         /// See NetLicensingAPI for details: https://netlicensing.io/wiki/licensee-services#validate-licensee
         /// </summary>
-        public static ValidationResult validate(Context context, String number, ValidationParameters validationParameters)
+        public static ValidationResult validate(Context context, String number, ValidationParameters validationParameters, int timeoutInSeconds = 100)
```

Comment:
Consider using TimeSpan instead of int

Reply:
I was considering it, but it is struct and it cannot be null. Maybe we can have a default TimeSpan for say 100 seconds which is the default on the HttpRequest